src/app/core/store/collections/collection.util.ts

```diff
@@ -75,3 +76,8 @@ export function createAttributesMap(attributes: Attribute[]): Record<string, Att
     return attributesMap;
   }, {});
 }
+
+export function findAttributeConstraint(attributes: Attribute[], attributeId: string): Constraint {
+  const attribute = (attributes || []).find(attr => attr.id === attributeId);
+  return attributes && attribute.constraint;
```

Comment:
There should be `attribute` and not `attributes` in the return statement.

Reply:
good catch!